compute/cloud-client/src/main/java/compute/CreateWithLocalSsd.java

```diff
@@ -0,0 +1,197 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute;
+
+// [START compute_instances_create_with_local_ssd]
+
+import com.google.cloud.compute.v1.AttachedDisk;
+import com.google.cloud.compute.v1.AttachedDiskInitializeParams;
+import com.google.cloud.compute.v1.Image;
+import com.google.cloud.compute.v1.ImagesClient;
+import com.google.cloud.compute.v1.Instance;
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.NetworkInterface;
+import com.google.cloud.compute.v1.Operation;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateWithLocalSsd {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // project: project ID or project number of the Cloud project you want to use.
+    String project = "your-project-id";
+    // zone: name of the zone to create the instance in. For example: "us-west3-b"
+    String zone = "zone-name";
+    // instanceName: name of the new virtual machine (VM) instance.
+    String instanceName = "instance-name";
+
+    createWithLocalSsd(project, zone, instanceName);
+  }
+
+  // Create a new VM instance with Debian 10 operating system and SSD local disk.
+  public static void createWithLocalSsd(String project, String zone, String instanceName)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+
+    // Get the latest debian image.
+    Image newestDebian = getImageFromFamily("debian-cloud", "debian-10");
+
+    // Create the disks to be included in the instance.
+    List<AttachedDisk> disks = new ArrayList<>();
+    String diskType = String.format("zones/%s/diskTypes/pd-standard", zone);
+    disks.add(diskFromImage(diskType, 10, true, newestDebian.getSelfLink(), true));
+    disks.add(localSsdDisk(zone));
+
+    // Create the instance.
+    Instance instance = createInstance(project, zone, instanceName, disks);
+
+    if (instance != null) {
+      System.out.printf("Instance created with local SSD: %s", instance.getName());
+    }
+
+  }
+
+  // Retrieve the newest image that is part of a given family in a project.
+  // Args:
+  //    project: project ID or project number of the Cloud project you want to get image from.
+  //    family: name of the image family you want to get image from.
+  private static Image getImageFromFamily(String project, String family) throws IOException {
```

Comment:
I would rename this explicitly to `projectId` or `projectNumber` (and anywhere else)

Reply:
I think this still needs to be renamed